docs/getting-started/first-service.md

```diff
@@ -44,6 +44,12 @@ You can also create a new repository by clicking on the **Create repository** bu
 
 ![](./assets/first-service/select-repository.png)
 
+:::note
+In addition to GitHub, Amplication also supports [Bitbucket](/sync-with-bitbucket) for Enterprise users.
```

Comment:
@dericksozo looks good. lets also mention that we support AWS CodeCommit

Reply:
Hey @GreenMachine01, I added that we support AWS CodeCommit. After the page is finished and published I'll make sure to link it here also.